guides/products-and-variants/overview.md

```diff
@@ -0,0 +1,85 @@
+# Overview of products and variants
+
+Products and variants are integral to Solidus. While the `Spree::Product` and
+`Spree::Variant` models depend on each other, it is important to understand how
+they are different:
+
+- `Spree::Product`s track the general information about a product. This includes
+  the product description and the permalink where a customer would find the
+  product listing on a store.
+- `Spree::Variant`s track the specific information about a variant of that
+  product. The variant stores the product price, dimensions, and weight. The
+  variant provides the information required by orders and shipments.
+
+The `Spree::Variant` model provides the price for each line item in an order.
+
+The rest of this article introduces essential information for using products and
+variants in Solidus.
+
+<!-- TODO:
+  It might be worth diagramming how Spree::Products, Spree::Variants,
+  and Spree::LineItems affect an order and how it's priced.
+-->
+
+## Products
+
+`Spree::Product`s track unique products within your store. If you sell a mug and
+a t-shirt, you would set up a product for each of them.
+
+If you have a number of items that are similar (like t-shirts that come in
+small, medium, and large sizes), you can create [variants](#variants) for a
+single product instead of creating three separate products.
+
+You can categorize products using [taxonomies and
+taxons](#taxonomies-and-taxons). And, if you want to offer more extensive
+information about a single product, you can add custom [product
+properties](product-properties.md) for any product.
+
+## Variants
+
+`Spree::Variant`s track the unique properties of multiple similar products that
+you sell. For example, if you sell a red mug and a green mug that have many
+other properties in common, you could create a single product ("Mug") with two
+variants.
+
+Here are a few key points to note about variants:
+
+- If a product has more than one variant, all variants require an option type
+  and option value. (For example, an option type of "Size" with the values
+  "Small", "Medium", and "Large".)
+- Every product has a master variant. When additional variants are created, they
```

Comment:
A master variant is a complicated concept that should have additional information.

Reply:
I added a link to more information about master variants here. It lives in the `variants.md` article. I've extended that section as well. 👍 Thank you.